app/helpers/application_helper.rb

```diff
@@ -116,6 +116,23 @@ def deeds_for(options={})
     render({ :partial => 'deed/deeds', :locals => { :limit => limit, :deeds => deeds, :options => options} })
   end
 
+  def show_prerender(prerender, locale) 
+    begin
+      prerenders = JSON.parse(prerender)
+      if prerenders.key? locale.to_s
```

Comment:
this could also be streamlined using assignment within a conditional:
```ruby
unless rendered = prerenders[locale.to_s]
  rendered = #fallback code here
end
# at this point rendered has the value
```